app/models/cqm/cql_library.rb

```diff
@@ -12,6 +12,11 @@ class CQLLibrary
     field :elm_annotations, type: Hash
     field :is_main_library, type: Boolean, default: false
 
+    # Currently this is only relevant for cql libraries that are included in composite measures,
+    # true indicates the library files are in the "top level" folder of the measure package, rather
+    # than coming from a component measure folder.
+    field :is_top_level, type: Boolean, default: true
```

Comment:
Isn't the field above, `:is_main_library` for this purpose already?

Reply:
No, I believe only one library can ever be the main library.

Reply:
Resolved on slack.